onnxruntime/python/tools/quantization/smooth_average.py

```diff
@@ -0,0 +1,151 @@
+# Computing a smooth average (weighted average of data split by change points)
```

Comment:
Can you add this as part of a separate PR... I have a bunch of comments here would be better if we separate this,

Reply:
Removed, and will add to a separate PR!